src/main/kotlin/no/nav/syfo/fastlege/api/system/access/APIConsumerAccessService.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.syfo.fastlege.api.system.access
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.security.token.support.core.context.TokenValidationContextHolder
+import no.nav.syfo.api.auth.OIDCUtil
+import no.nav.syfo.util.configuredJacksonMapper
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import javax.ws.rs.ForbiddenException
+
+@Component
+class APIConsumerAccessService(
+    private val tokenValidationContextHolder: TokenValidationContextHolder,
+    @Value("\${azure.app.pre.authorized.apps}") private val azureAppPreAuthorizedApps: String,
+) {
+    private val preAuthorizedClientList: List<PreAuthorizedClient> =
+        configuredJacksonMapper().readValue(azureAppPreAuthorizedApps)
+
+    fun validateConsumerApplicationAZP(authorizedApplicationNameList: List<String>) {
```

Comment:
Pirk! Har googla meg frem til at AZP står for authorized party, kanskje det kan skrives Azp?

Reply:
Kan godt det, endret til `Azp` nå.